src/app/Library/Validation/Rules/ValidArray.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Backpack\CRUD\app\Library\Validation\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\Rule;
+use Illuminate\Contracts\Validation\ValidationRule;
+use Illuminate\Support\Facades\Validator;
+
+abstract class ValidArray extends BackpackCustomRule
+{
+    public array $itemRules = [];
+
+    public array $namedItemRules = [];
```

Comment:
```php
// itemRules
'array.*' => [general array rules]

// namedItemRules
'array.name' => [one rule],
'array.description' => 'other rule'
```

I guess it stood to you because they are only used in ValidNestedArray, I will move them there. 👍